src/cclib/io/moldenwriter.py

```diff
@@ -16,22 +16,35 @@
 class MOLDEN(filewriter.Writer):
     """A writer for MOLDEN files."""
 
+    def __init__(self, ccdata, *args, **kwargs):
+        """Initialize the MOLDEN writer object.
+
+        Inputs:
+          ccdata - An instance of ccData, parsed from a logfile.
+        """
+        # Call the __init__ method of the superclass
+        super(MOLDEN, self).__init__(ccdata, *args, **kwargs)
+
+        required_attrs = ['atomcoords', 'atomnos', 'natom']
```

Comment:
Doesn't matter to me if it's a variable or constant (there are no constants in Python), but a something attached to the class, not the object.